opencl/source/sharings/gl/linux/gl_sync_event_linux.cpp

```diff
@@ -27,10 +27,13 @@ GlSyncEvent::GlSyncEvent(Context &context, const GL_CL_SYNC_INFO &sync)
 }
 
 GlSyncEvent::~GlSyncEvent() {
+#if 0
```

Comment:
same in this file

Reply:
Although here we should have something to properly support sync point, the current code in there doesn't work (just crash calling to nullptr) so it's removed until someone implements it properly.